med-inventory/src/components/purchase-medicine/purchase-layout/numericEditor.jsx

```diff
@@ -0,0 +1,138 @@
+import React, {Component} from "react";
+
+const KEY_BACKSPACE = 8;
+const KEY_DELETE = 46;
+const KEY_F2 = 113;
+
+export default class NumericEditor extends Component {
+    constructor(props) {
+        super(props);
+
+        this.cancelBeforeStart = this.props.charPress && ("1234567890".indexOf(this.props.charPress) < 0);
```

Comment:
Please add comment explaining the magic string "1234567890"

Reply:
ok